service/share/interface.go

```diff
@@ -12,7 +12,7 @@ var ErrNotAvailable = errors.New("da: data not available")
 // AvailabilityTimeout specifies timeout for DA validation during which data have to be found on the network,
 // otherwise ErrNotAvailable is fired.
 // TODO: https://github.com/celestiaorg/celestia-node/issues/10
-const AvailabilityTimeout = 10 * time.Minute
+const AvailabilityTimeout = 20 * time.Minute
```

Comment:
why did you chose to double this?

Reply:
It can be the same if I make RetrieveQuadrantTimeout = 2.5 min